app/src/main/java/org/room76/apollo/rooms/RoomsFragment.java

```diff
@@ -175,19 +181,42 @@ public RoomsAdapter(List<Room> rooms, RoomItemListener itemListener) {
 
         @Override
         public ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
```

Comment:
WeakReference is cool, but you can simply get context from parent - parent.getContext()

Reply:
Not in *onBindViewHolder* method. The question of what is worse -- get context from child of View of ViewHolder or saving it in WeakRef in *onCreateView* -- is open.

Reply:
parent.getContext() will return actual application context